livre/core/data/NodeId.cpp

```diff
@@ -30,30 +30,26 @@ NodeId::NodeId( const Identifier& identifier )
     : _id( identifier )
 {}
 
-NodeId::NodeId( const uint32_t level,
-                const Vector3ui& position,
+NodeId::NodeId( const uint32_t level, const Vector3ui& position,
```

Comment:
It's in a separate cleanup commit. review the commits individually if you want small changes.